### ISSUE TITLE
Oversized Hats

### ISSUE THREAD

**Issue by ghost** [deferred-impact/battledudes-issue-tracker#231]
Some hats can hide a person's hands, when they cook a grenade.

To be specific, I mean hats which extend a little over a character's body. 
The grenade is barely visible, due to the hat. 
Now if you were to use a sticky grenade, which is even smaller, it would be worse. 
[Screen recording 2023-10-18 7.03.49 PM.webm](https://github.com/deferred-impact/battledudes-issue-tracker/assets/148166198/4393a4da-8dc3-47b5-994b-7e4251930166)

**Comment by Haox09** [comment]
It's intended

**Comment by deferred-impact** [comment]
If it is intended, it certainly shouldn't be.

I would argue, however, that the better solution here would be to create a separate icon for a cooked nade. Even without a hat, sometimes it's just impossible to tell if a grenade is being cooked, and if so, which type.

**Comment by ghost** [comment]
I believe the best idea would be to keep hats from hiding grenades, by simply keeping them from towering over your hands, just rework them to be smaller in terms of the front. 

> I would argue, however, that the better solution here would be to create a separate icon for a cooked nade. 

I don't know what and how exactly a 'separate icon' would work in terms of alerting other players about grenades. 

> Even without a hat, sometimes it's just impossible to tell if a grenade is being cooked, and if so, which type.

I think you are referring to offscreen nades, which are quite an issue. However, if the range of frag was to be nerfed, it would be underpowered. Maybe sticky grenades's range  could be reduced, because it is quite far, compared to frag.

**Comment by deferred-impact** [comment]
> I believe the best idea would be to keep hats from hiding grenades, by simply keeping them from towering over your hands, just rework them to be smaller in terms of the front.

Redrawing dozens of hats? Good luck convincing the developer to do that

> I don't know what and how exactly a 'separate icon' would work in terms of alerting other players about grenades.

An icon hovering near a player who is cooking the grenade. Layered on top of the skin layer so even if there is overlap, the icon is still visible, drawn on top of the skin

> I think you are referring to offscreen nades, which are quite an issue. However, if the range of frag was to be nerfed, it would be underpowered. Maybe sticky grenades's range could be reduced, because it is quite far, compared to frag.

No, i'm referring to not being able to properly see someone cooking a frag when they are on my screen. Even if they are wearing no hat. Some gloves blend in with grenades too well, for example

**Comment by ghost** [comment]
> Redrawing dozens of hats? Good luck convincing the developer to do that

I don't believe there's that many, there's like 6. Besides, that's not the dev's job, its the artist's job. 

> An icon hovering near a player who is cooking the grenade. Layered on top of the skin layer so even if there is overlap, the icon is still visible, drawn on top of the skin

Good luck getting the dev to add that. 

> No, i'm referring to not being able to properly see someone cooking a frag when they are on my screen. Even if they are wearing no hat. Some gloves blend in with grenades too well, for example

Which gloves? For example what? I don't think there''s any gloves that do that.
